Makefile.Microsoft_nmake

```diff
@@ -0,0 +1,94 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CFLAGS = /nologo /O2 /Imlkem /Ifips202 /Ifips202/native /Imlkem/sys /Imlkem/native
```

Comment:
Does this build opt or non-opt, and (how) can one switch between the two?

Reply:
I currently only compile non-opt code, as I don't know how to compile the assembly files on windows.

For example I get a bunch of errors like
```
.\mlkem\native\x86_64\basemul.S(1) : error A2008:syntax error : /
.\mlkem\native\x86_64\basemul.S(3) : error A2008:syntax error : /
.\mlkem\native\x86_64\basemul.S(4) : error A2008:syntax error : /
.\mlkem\native\x86_64\basemul.S(6) : error A2044:invalid character in file
.\mlkem\native\x86_64\basemul.S(7) : error A2044:invalid character in file
.\mlkem\native\x86_64\basemul.S(8) : error A2044:invalid character in file
.\mlkem\native\x86_64\basemul.S(9) : error A2044:invalid character in file
...
```
when trying to compile assembly files to object file by
```
ml /c .\mlkem\native\x86_64\basemul.S
```

Reply:
Let's restrict to the non-opt implementation for this PR - please write it somewhere as a comment in the Makefile. 
I'll try to find someone who can help us with getting the assembly implementations to work.

Reply:
Works for me, but it should be commented on in the Makefile, README, and commit message.